student.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'person'
+
+class Student < Person
+  def initialize(age, classroom, name = 'Unknown', parent_permission: true)
+    @classroom = classroom
+    super(name, age, parent_permission: parent_permission)
```

Comment:
- [ ] Hello, Kindly as mentioned by the previous reviewer, look at the order of parameters in the super and pay attention to parameters with default options

```suggestion
    # *Inherits from Person. Link to the inheritance example in a previous lesson https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_principles_examples.md#inheritance.
class Student < Person 
  # *Constructor extends parent's constructor by adding @classroom and a parameter for it. Link to the example about constructors in a previous lesson https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/class_and_object_for_irb.md#constructor-and-instance-vars
  def initialize(age, classroom, name = "Unknown", parent_permission = true)  
    super(age, name, parent_permission)
    @classroom = classroom
  end
  # *Method play_hooky that returns "¯\(ツ)/¯".
  def play_hooky
    "¯\(ツ)/¯"
  end
end

```